packages/auth/lib/index.d.ts

```diff
@@ -1871,6 +1871,19 @@ export namespace FirebaseAuthTypes {
      * @param url: emulator URL, must have host and port (eg, 'http://localhost:9099')
      */
     useEmulator(url: string): void;
+    /**
+     * Provides a MultiFactorResolver suitable for completion of a multi-factor flow.
+     *
+     * @param error: The MultiFactorError raised during a sign-in, or reauthentication operation.
+     */
+    getMultiFactorResolver(error: MultiFactorError): MultiFactorResolver;
```

Comment:
I'm curious about your opinion. Should this be typed as `unknown` just the way Typescript types exceptions in the catch-clause, or do we want to make the user check that the exception fits a `MultiFactorError` when calling?

Reply:
I want it to be typed exactly the same as it is in firebase-js-sdk (allowing for the v8/v9 style difference from previous comments) - and this was copy/pasted from their docs, so, I think this is the right way to go?

Reply:
Looked it up. This is the exact typing `firebase-js-sdk` uses. 👍